ext/hal/nxp/imx/drivers/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+zephyr_include_directories(.)
+
+zephyr_sources(clock_freq.c)
+zephyr_sources_ifdef(CONFIG_CLOCK_CONTROL_IMX_CCM   ccm_imx7d.c)
+zephyr_sources_ifdef(CONFIG_CLOCK_CONTROL_IMX_CCM   ccm_analog_imx7d.c)
```

Comment:
Should be also constrained by CONFIG_SOC_MCIMX7_M4, as CONFIG_SOC_MCIMX6X_M4 would add ccm_imx6sx.c and ccm_analog_imx6sx.c files instead.

Reply:
Agreed.